classes/Tools.php

```diff
@@ -3035,11 +3035,9 @@ public static function getDefaultIndexContent()
     /**
      * Return the directory list from the given $path.
      *
-     * @param string $path
-     *
-     * @return array
+     * @return array<string>
      */
-    public static function getDirectories($path)
+    public static function getDirectories(string $path): array
```

Comment:
We  can't change signatures of public functions

Reply:
Makes sense, I will revert the signature changes

Reply:
done